### PR TITLE
Update cache.md to include Homebrew plugin and template locations

### PR DESCRIPTION
**Pull request by tpmccallum**
Signed-off-by: tpmccallum tim.mccallum@fermyon.com

### CODE REVIEW

File: content/spin/cache.md
Comment (itowlson) [inline content/spin/cache.md:39]: I think this would be `$HOMEBREW_PREFIX/fermyon-spin/plugins`... @kate-goldenring?

File: content/spin/cache.md
Comment (itowlson) [inline content/spin/cache.md:47]: Similarly I think this would have a `templates` suffix

**Review by itowlson — COMMENTED** [review]
Wording looks fine but needs @kate-goldenring's signoff on the paths.

Is there particular reason not to do this as part of #818?  It doesn't much matter of course, if this is more convenient then no worries.

File: content/spin/cache.md
Comment (itowlson) [inline content/spin/cache.md:39]: Oh sorry missed this
```suggestion
> Note: If you [install Spin](install) using Homebrew, the plugins are stored at `$HOMEBREW_PREFIX/fermyon-spin`.
```
(or remove the square brackets)

File: content/spin/cache.md
Comment (itowlson) [inline content/spin/cache.md:47]:
```suggestion
> Note: If you [install Spin](install) using Homebrew, the templates are stored at `$HOMEBREW_PREFIX/fermyon-spin`.
```

**Review by itowlson — CHANGES_REQUESTED** [review]
Sorry missed this before - there's some things that look like they're intended to be links?

File: content/spin/cache.md
Comment (tpmccallum) [reply content/spin/cache.md]: I am having a bit of difficulty finding the plugins and templates directory.
I am using the following command to locate where Spin installed:

```bash
$ brew --prefix fermyon/tap/spin
/usr/local/opt/spin
```

But there might be a better way of doing this.

**Comment by tpmccallum** [comment]
Good catch (on the square brackets) @itowlson 
@kate-goldenring I am still curious about how a user can find their templates and plugins (or the brew datadir where Spin exists). The documentation to upgrade Spin makes the assumption that a user may have forgotten how they installed Spin in the first place and we try and help them by providing a few OS-specific command like `which spin` for Linux etc.

If I use:

```bash
brew --prefix fermyon/tap/spin
/usr/local/opt/spin
```
I don't see any templates or plugins directories in there. (Prefix all of this with ... I am probably doing it wrong.) :)

**Comment by itowlson** [comment]
Do you have a `$HOMEBREW_PREFIX` environment variable?

**Review by kate-goldenring — APPROVED** [review]

File: content/spin/cache.md
Comment (kate-goldenring) [inline content/spin/cache.md:39]:
```suggestion
> Note: If you [install Spin](install) using Homebrew, the plugins are stored at `$HOMEBREW_PREFIX/fermyon-spin/plugins`.
```

File: content/spin/cache.md
Comment (kate-goldenring) [inline content/spin/cache.md:47]:
```suggestion
> Note: If you [install Spin](install) using Homebrew, the templates are stored at `$HOMEBREW_PREFIX/fermyon-spin/templates`.
```

**Review by kate-goldenring — CHANGES_REQUESTED** [review]

**Comment by kate-goldenring** [comment]
@tpmccallum `echo $HOMEBREW_PREFIX ` is synonymous with `brew --prefix`. `brew --prefix fermyon/tap/spin` is a different directory.

**Comment by tpmccallum** [comment]
Thanks @kate-goldenring 
On my system (installing without first setting the `$HOMEBREW_PREFIX` to be  synonymous with `brew --prefix`) it seems to have installed Spin inside an additional `etc` dir.

```bash
/usr/local/etc/fermyon-spin
```

**Comment by tpmccallum** [comment]
Just listing paths, below, FYI.

```bash
$ brew --prefix
/usr/local
$ brew --prefix fermyon/tap/spin
/usr/local/opt/spin
```

**Comment by tpmccallum** [comment]
```bash
$ pwd
/usr/local/etc/fermyon-spin/templates
```

```bash
$ ls
http-c_c328011daf311449f512a99cc0e60bbcae479c2b9d0792609b69834a0aec067f

// --snip --

static-fileserver_128e8f24176db25f32ae793c897d5f3af60349022892ab1e1a4f0b22b04c563e
```

**Comment by tpmccallum** [comment]
I guess this is just a local environment issue (of the user) because everything inside `fermyon-spin` is correct; making these docs correct. Happy to merge if approved.
Thanks for the comments above.

**Review by itowlson — APPROVED** [review]

**Review by itowlson — APPROVED** [review]
wording looks fine, deferring to Kate for accuracy though!

**Comment by kate-goldenring** [comment]
@tpmccallum the plugins are installed into a different directory than the spin binary. That is expected because during upgrades, the folder of the spin binary is deleted and replaced. We dont want all plugins to be deleted and replaced with the defaults every upgrade; rather, we want those to persist (say if i have installed a custom plugin)

**Review by kate-goldenring — APPROVED** [review]

**Pull request merged**
